jobsdb/jobsdb.go

```diff
@@ -1893,7 +1893,7 @@ func (jd *HandleT) getFileUploader() (filemanager.FileManager, error) {
 	}
 	return filemanager.New(&filemanager.SettingsT{
 		Provider: config.GetEnv("JOBS_BACKUP_STORAGE_PROVIDER", "S3"),
-		Config:   filemanager.GetProviderConfigFromEnv(),
+		Config:   filemanager.GetProviderConfigFromEnv("JOBS_BACKUP"),
```

Comment:
Why do we need this new signature? This is the only function that seems to be using it.

Reply:
Now the failed jobs get dumped from the new jobsdb into a different provider. Look for PROC_ERROR_BACKUP in processor.go